packages/manager/apps/pci-object-storage/src/pages/object-storage/storage/s3Id/dashboard/Dashboard.page.tsx

```diff
@@ -4,12 +4,16 @@ import { Outlet } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import BucketOverview from './_components/BucketOverview.components';
 import Tags from './_components/Tags.component';
+import Guides from '@/components/guides/Guides.component';
 
 const Dashboard = () => {
   const { t } = useTranslation('pci-object-storage/storages/s3/dashboard');
   return (
     <>
-      <h2>{t('dashboardTitle')}</h2>
+      <div className="flex justify-between w-full items-center">
```

Comment:
Maybe this piece of code could be refactored? Because I've seen it several times on the PR. Whatever you think is best!